pkg/cloudprovider/provider/vmwareclouddirector/provider.go

```diff
@@ -328,6 +329,12 @@ func (p *provider) getConfig(provSpec clusterv1alpha1.ProviderSpec) (*Config, *p
 	}
 
 	c := Config{}
+
+	c.APIToken, err = p.configVarResolver.GetConfigVarStringValueOrEnv(rawConfig.APIToken, "VCD_API_TOKEN")
+	if err != nil {
+		return nil, nil, nil, fmt.Errorf("failed to get the value of \"apiToken\" field, error = %w", err)
```

Comment:
```suggestion
		return nil, nil, nil, fmt.Errorf(`failed to get the value of "apiToken" field, error = %w`, err)
```

Just an example how to avoid escaping.